docs/source/swift-scripting.md

```diff
@@ -124,87 +123,84 @@ One of the convenience wrappers available to you in the target is `ApolloSchemaD
+5. Build and run using the Xcode project. Note that if you're on Catalina you might get a warning asking if your executable can access files in a particular folder like this:
 
    ![permission prompt](screenshot/would_like_to_access.png)
    
-   Click the "OK" button. Your CLI output will look something like this: 
+   Click **OK**. Your CLI output will look something like this: 
    
-   ![log barf for successful run](screenshot/schema_download_success.png)
+   ![log output for successful run](screenshot/schema_download_success.png)
    
-   Those last two lines - "Saving schema started" and "Saving schema completed" indicate that the schema has successfully downloaded. 
+   The last two lines  (`Saving schema started` and `Saving schema completed`) indicate that the schema has successfully downloaded. 
+
+Note the warning: This isn't relevant for schema downloading, but it *is* relevant for generating code: In order to generate code, you need both the schema and some kind of operation.
 
-Note the warning: This isn't relevant for schema downloading, but it *is* relevant for generating code: In order to generate code, you need both the schema and some kind of operation. Now that you've got the schema, it's time to 
+## Using codegen to create a `.graphql` file with an operation
 
-## Codegen theory + creating a `.graphql` file with an operation
+Code generation requires both of the following to run:
 
-Code generation takes a combination of the **schema**, which defines what it's *possible* for you to request from or send to your server, and your **operations**, which define what you are *actually* requesting from the server. 
+* Your **schema**, which defines what it's *possible* for you to request from or send to your server
+* A collection of **operations**, which define what you are *actually* requesting from the server
 
-An operation can be one of three things: 
+Each operation can be one of three things: 
 
 - A **query**, which is a one-time request for specific data
 - A **mutation**, which changes data on the server and then receives updated data back
 - A **subscription**, which allows you to listen for changes to a particular object or type of object
 
-The code generation takes your operations and compares them to the schema to validate that what you are asking for is, in fact, possible. If it's not possible, the whole process errors out. If it is possible, it generates Swift code that gives you end-to-end type safety for each operation. 
-
-Thus, a simple equation can be used to describe generating code: 
-
-`schema + operations = code`
-
-If you're missing either of the first two parts, the code can't be generated. If there's operations but no schema, the operations can't be validated, so we can't know if code should be generated. If there's a schema but no operations, then there's nothing to validate or generate code for. 
```

Comment:
I would like to re-add this paragraph and the `schema + operations = code` bit, since it explains *why* this doesn't work if you don't have both, which a lot of people gloss over in my experience.